### ISSUE TITLE
Modifier keys doesn't stick on double press

### ISSUE THREAD

**Issue by iprok** [UltimateHackingKeyboard/firmware#150]
I have upgraded to firmware 8.4.2 and I've just notified that not mouse nor Mod key doesn't stick anymore when double pressing it.

**Comment by mondalaci** [comment]
Chances are you have to double tap more quickly. Please give it a try.

Which was your previous firmware version?

**Comment by kareltucek** [comment]
(The same problem here - if I double tap extremely fast, it does lock. However, the timeout is set so low that I am almost unable to do it in practice. No chance to activate lock with a standard doubletap.)

**Comment by mondalaci** [comment]
Ultimately, we'll make the double tap to lock timeout configurable via Agent but it'll take some time because it'll involve changing the configuration format, and increasing the hardcoded timeout would make people accidentally lock Mod, which would mostly do a disservice usability-wise.

As the first step, Agent will make it possible to configure the the double tap to lock feature on a per key basis. This is however dependent on #148, so we have to fix it first. Afterwards, we can increase the hardcoded timeout, and eventually, it'll be configurable via Agent.

Closing this, as it's a redundant issue.

**Comment by mhantsch** [comment]
I guess this is a duplicate of https://github.com/UltimateHackingKeyboard/agent/issues/749. Putting this comment in here for easier reference to that issue.
